modules/richaudienceBidAdapter.js

```diff
@@ -197,6 +199,10 @@ function raiGetVideoInfo(bid) {
       playerSize: bid.mediaTypes.video.playerSize,
       mimes: bid.mediaTypes.video.mimes
     };
+  } else {
+    videoData = {
+      format: 'banner'
```

Comment:
Seems strange to be defining a video bid as 'banner', shouldn't this be checking the video.context for 'outstream' then apply that to the 'format'? You would then check against that in line 107. Maybe there is an edge case I am missing?

Reply:
The definition is correct and it’s done willingly, we’re able to serve video demand on banner spaces if the final partner wishes to. That’s managed from our internal platform, flagging the placements as multi-format. The adaptor must be able to support both formats as well.

Reply:
That makes sense and is defined as Oustream within Prebid. You have a conditional on line 107 that looks for the videoData.format === 'outstream' but you are not setting this value anywhere, it will always fail. It should be looking at the bidRequest.mediaType.video.context: https://docs.prebid.org/dev-docs/bidder-adaptor.html#ingesting-the-video-context

Reply:
Hey Niko,
That’s a quick scheme on how are we going to use the new param info:
Config type — demand type — expected render

video -> video -> video
video -> banner -> none
banner -> banner -> banner
banner -> video -> video (If the publisher mismatches the config, this case is a safe check)

![video-video](https://user-images.githubusercontent.com/26602871/111602103-517e1a00-87d3-11eb-80a2-e8516bdf5e2c.png)
![video-banner](https://user-images.githubusercontent.com/26602871/111602114-5347dd80-87d3-11eb-91d3-0940e8a21e45.png)
![banner-banner](https://user-images.githubusercontent.com/26602871/111602116-53e07400-87d3-11eb-8e56-77ee4a2a3f12.png)
![banner-video](https://user-images.githubusercontent.com/26602871/111602120-5511a100-87d3-11eb-83b8-6a3193ceaeb5.png)

Cheers

Reply:
Thank you for the explanation and for adding the additional fields to your Renderer!